### DISCUSSION TITLE
Moving entries between organizations

### DISCUSSION THREAD

**Discussion by sightkick** [dani-garcia/vaultwarden#3353] [General]
I have 2 organizations which I own in vaultwarden (Server V1.27) and I am not able to move one or more entries from one of there to the other organization Vault. If I set an entry in "my vault" then I am able to move it to any of the organizations which I own. What´s the problem?

<img width="497" alt="grafik" src="https://user-images.githubusercontent.com/57866026/226118867-e2ca0b0c-7544-465e-8044-113e70f4dec1.png">

**Comment by BlackDex** [comment]
You can't move/clone/copy organization items to an other organization from the main (personal) vault view.
You can only move personal items.

From the organization vault view you are able to clone an item to your personal vault or an other organization you have access to.
The organization vault view is only accessible for managers or higher.

**Reply by sightkick** [reply]
OMG.... that´s what I´se seen sometimes but I never found it again. THANKS a lot. But maybe this should be a feature request I have to rise: Possibility to move an entry to another organization from the personal view if the status is manager or higher from both orgs? I don´t even understand why there ist this second view of vault items under organizations... but I mention it´s a feature and not a bug? I don´t understand...

**Reply by BlackDex** [reply]
There is a difference. Admins or Owners have access to all items.
This could clutter there personal view by default.

You can configure collections for admins and owners and mark just one or two as visible. That would cause all the items in other collections to not being visible on the personal vault overview, but they will be in the organization view.

**Reply by sightkick** [reply]
Thanks for your quick reply. I think I have to learn how to use the structure of my vaultwarden as an msp. To understand this structure and possibilities ist very hard. Organizations > Collections > Folders.... different view levels. Is there any document to learn this in an easy way? I read a lot of articles but I´m not getting smarter in any way.

**Comment by BlackDex** [comment]
I would suggest to check here https://bitwarden.com/help/getting-started-webvault/

**Comment by sightkick** [comment]
Thanks a lot. Are you a developer in contribution to the "main" vaultwarden from dani garcia??

**Reply by BlackDex** [reply]
Yes 😄
